linux/power_operation_scripts/set_power_script_custom.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check customized power operation script
+# # Parameters:
+#   power_cmd_op: power operation
+# # Return:
+#   script_file_path: path of customized script file
+#   script_tag_path: path of tag file created by customized script file
+#   script_file_in_conf_file: the script file in tools.conf
+
+- name: "Set fact of customized script file"
+  ansible.builtin.set_fact:
+    script_file_path: "/var/{{ power_cmd_op }}-vm-custom"
+    script_tag_path: "/var/{{ power_cmd_op }}.tag"
+    script_file_in_conf_file: "/var/{{ power_cmd_op }}-vm-custom"
+
+- name: "Remove the existing script and tag files for {{ power_cmd_op }} script"
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ script_file_path }}"
+    - "{{ script_tag_path }}"
+  delegate_to: "{{ vm_guest_ip }}"
+  ignore_errors: true
+
+- name: "Create script file for {{ power_cmd_op }} script"
```

Comment:
These 2 tasks can be merged to 1 using "ansible.builtin.lineinfile" with 'create' and 'mode' parameters.

Reply:
Need to remove 2 files and only create one file.
So it always needs two tasks to finish it.  


Thanks.